extensions/cornerstone/src/init.js

```diff
@@ -102,6 +102,32 @@ export default function init({ servicesManager, configuration }) {
     csTools.BrushTool,
   ];
 
+  /* Measurement Service Events */
+  cornerstone.events.addEventListener(
+    cornerstone.EVENTS.ELEMENT_ENABLED,
+    event => {
+      MeasurementService.subscribe(
+        /* Should subscription event names be handled outside like here? Where to keep available subscription event names? */
+        csTools.EVENTS.MEASUREMENT_ADDED,
+        updatedAnnotation =>
+          console.log(
+            '[subscribe] Measurement added or updated',
+            updatedAnnotation
+          )
+      );
+      const element = event.detail.element;
+      element.addEventListener(csTools.EVENTS.MEASUREMENT_ADDED, event => {
+        console.log('[addOrUpdate] Adding new measurement...', event.detail);
+        MeasurementService.addOrUpdate({
+          id: 1,
+          annotation: event.detail,
+          /* Should subscription event names be handled outside like here? Where to keep available subscription event names? */
+          broadcastEventName: csTools.EVENTS.MEASUREMENT_ADDED,
```

Comment:
I don't think we need to provide this. It might be worth telling the MeasurementService "who" is pushing the change (ie. cornerstone), then maybe we could ignore subscriptions/listeners for it because it should already have the most up-to-date info?)

Reply:
Updated.